src/main.c

```diff
@@ -40,15 +40,26 @@
 // In case there is an api change, first *verify* changes
 // (especially potential security implications) before bumping
 // the API level!
+#if defined(TARGET_NANOS)
 STATIC_ASSERT(CX_APILEVEL == 9, "bad api level");
+#elif defined(TARGET_NANOX)
+STATIC_ASSERT(CX_APILEVEL == 10, "bad api level");
+#endif
 
 // These are global variables declared in ux.h. They can't be defined there
 // because multiple files include ux.h; they need to be defined in exactly one
 // place. See ux.h for their descriptions.
 // To save memory, we store all the context types in a single global union,
 // taking advantage of the fact that only one command is executed at a time.
 
+#if defined(TARGET_NANOS)
```

Comment:
Maybe we should move this to uiHelpers?

Reply:
it would be nice, but I don't know how to achieve making only one global instance of the variables, as suggested in the comments above